### PR TITLE
MKT-1193 -- Opties Application Page

### PR DESCRIPTION
**Pull request by stewsmith**
[MKT-1193](https://optimizely.atlassian.net/browse/MKT-1193)

### CODE REVIEW

**Comment by dtothefp** [comment]
@stewsmith looks good, this form could probably just have a width set and be `margin: 0 auto` but doesn't really matter that much

screen shot 2015-03-19 at 12 10 01 pm

**Comment by stewsmith** [comment]
@dtothefp -- re above: the padding is a results of the columns class. No need to mess with it I think.

**Comment by dtothefp** [comment]
@stewsmith re: above. The offset isn't a result of columns class it's a result of `offset-two`. You could take that off and do `margin: 0 auto` and it would center.

**Comment by dtothefp** [comment]
@stewsmith layout is a bit busted up in tablet width in the top panel, and padding on left for columns below seems to be non-existent

screen shot 2015-03-23 at 5 38 02 pm

**Comment by dtothefp** [comment]
@stewsmith `contact-partner-manager` link smooth scrolls to form when it should send email

**Comment by dtothefp** [comment]
@stewsmith probably throw some padding on the bottom of the `register now` button in the mobile nav

screen shot 2015-03-23 at 5 45 17 pm

File: website-guts/assets/css/pages/opticon-opties-success.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+@media only screen and (min-width: $desktop-min-width) {
+  body.opticon-opties-success {
+    .description-text {
+      width: 75%;
+      padding: 0 0 300px 0;
+    }
+    .confetti-container .confetti-right {
+      display: block;
+    }
```
Comment (dtothefp) [inline website-guts/assets/css/pages/opticon-opties-success.scss]: @stewsmith probably don't need the double specificity here, `.confetti-right` would probably work

File: website-guts/assets/js/pages/opties.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+$('.js-category-cta').on('click', function(event) {
+  var category = ($(this).data().category);
+  $('#Field1').val(category);
+  w.optly.mrkt.utils.smoothScroll.apply(this, [event]);
+});
```
Comment (dtothefp) [inline website-guts/assets/js/pages/opties.js]: @stewsmith hipster shit. this `$(this).data().category` can also be `$(this).data('category')` I think and you also don't need the wrapping parens

File: website/opticon/opties/opties_data.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      cta: "APPLY →"
+    -
+      title: "Data-Driven Culture Award"
+      blurb: "Has experience optimization transformed the way your team makes decisions? This Optie will be awarded to the company who has implemented the strongest optimization culture across the organization. Tell us about an increase in productivity and engagement, forward-thinking and data-driven teams, or increased organizational readiness for tackling goals."
+      cta: "APPLY →"
+    -
+      title: "Optimization Champion of the Year"
+      blurb: "Do you or one of your colleagues exemplify testing culture at your organization? This Optie will be awarded to an individual who embodies the impactful transformation enabled by experience optimization. He/she will have demonstrated an appetite for increasing their technical know-how and driving organizational change through data, while also boosting their personal career path."
+      cta: "APPLY →"
```
Comment (dtothefp) [inline website/opticon/opties/opties_data.yml]: @stewsmith it doesn't matter for this page but in the future we should try to keep symbols out of YML for tranlation purposes.  The arrow could easily be in the hbs `{{cta}}  →`

File: website/opticon/opties/success/opties_success.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+page_title: "Thank you!"
+blurb: "Don't stop now, submit a nomination for more <a href='https://www.optimizely.com/opticon/opties'>Optie Awards</a>!
+  <br>We will reach out in May to notify if you are a finalist. Good luck!"
```
Comment (dtothefp) [inline website/opticon/opties/success/opties_success.yml]: @stewsmith you should move this link out of the YML and into HBS with `linkPath`

File: website-guts/assets/css/pages/opticon-opties-success.scss (hunk 1 above)
Comment (stewsmith) [inline website-guts/assets/css/pages/opticon-opties-success.scss]: Double specificity is needed to override `confetti-right` in `css/layouts/opticon.scss`.  This success page is special in that it looks crowded when the confetti is there on tablet.

File: website-guts/assets/js/pages/opties.js (hunk 2 above)
Comment (stewsmith) [inline website-guts/assets/js/pages/opties.js]: Nice. Changed.

File: website/opticon/opties/opties_data.yml (hunk 3 above)
Comment (stewsmith) [inline website/opticon/opties/opties_data.yml]: :+1:

**Comment by dtothefp** [comment]
@stewsmith LGTM 

double check in prod that the form redirect goes to the right place, right now it goes to a staging branch link. Not sure how this works?

**Comment by stewsmith** [comment]
@dtothefp Yup the redirect happens on wufoo's side. Changing that once we merge.

**Pull request merged**